### DISCUSSION TITLE
Background color of area with cards is not customizable.

### DISCUSSION THREAD

**Discussion by traviskirton** [vimeo/player.js#986] [Bug reports]
### Expected Behavior
When loading an iframe, setting the `?transparent=0` variable should apply to the background element of the space for cards.

Per [vimeo documentation](https://vimeo.zendesk.com/hc/en-us/articles/115014089487-Iframe-background-color-for-embeds), the following code should make the iframe transparent:

```
<iframe src="https://player.vimeo.com/video/137805268?transparent=0" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>'
```

I expect that when there are cards on a video, these too should be shown with a transparent background, like the following image.

<img width="350" alt="Screen Shot 2020-01-03 at 10 14 22 AM" src="https://user-images.githubusercontent.com/1447498/71740816-0ff19100-2e12-11ea-9ba9-7fe9e1710ed8.png">

Similarly, if I set the style of an iframe to have a white background color, this should also affect the background area of the cards.

I expect the following div to have a customizable background color:

`div.vp-cards-wrapper.cards-wrapper--withThumbnails`

<img width="777" alt="Screen Shot 2020-01-03 at 10 30 57 AM" src="https://user-images.githubusercontent.com/1447498/71741649-4d571e00-2e14-11ea-93c2-d968d994e570.png">

### Actual Behavior
The iframe is transparent, but the cards area is NOT transparent. 

<img width="368" alt="Screen Shot 2020-01-03 at 10 13 42 AM" src="https://user-images.githubusercontent.com/1447498/71740831-20097080-2e12-11ea-8568-d878960c34c1.png">

### Steps to Reproduce

From your browser, go to:

https://createwithflow.com/learn/get-up-and-running/

Inspect the page, then make the preview area small enough to trigger the mobile-sized version of the page. 

By going into the inspector, and finding the proper div, I can toggle the background color on / off. 

![Screen-Recording](https://user-images.githubusercontent.com/1447498/71741436-c609aa80-2e13-11ea-8597-36cadfb341af.gif)

**Comment by luwes** [comment]
thanks for the detailed request, we will look into this.
